components/Layout/FormStyles/StyledLabel/index.js

```diff
@@ -4,7 +4,7 @@ import styled from "styled-components";
 const StyledLabel = styled.label`
   display: flex;
   flex-direction: column;
-  margin-bottom: 8px;
+  margin-bottom: 0px;
```

Comment:
When specifying a value of 0 for any property, the unit (like px, em, %, etc.) is not required because 0 is the same in any measurement unit. 

```suggestion
  margin-bottom: 0;
```